kingdom-of-ocelader/TrestlebridgeFarms
# Av chicken house

# Description

Create chicken house, purchase chicken (still not able to place in chicken house).

Applies to issue # 4


# Testing Instructions

Start.
Create Facility => Chicken House
Purchase Stock => Chicken

Exception will be thrown when attempting to add chicken to chicken house, this is expected and will be fixed in a future issue.
